aspnet/migration/rc1-to-rc2.rst

```diff
@@ -76,15 +121,28 @@ IHostingEnvironment changes
 
 All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
 
-======================  ===========================
-Old prefix              New prefix
-======================  ===========================
-ASPNET\_WEBROOT         ASPNETCORE\_WEBROOT
-ASPNET\_SERVER          ASPNETCORE\_SERVER
-ASPNET\_APP             ASPNETCORE\_APPLICATIONNAME
-ASPNET\_ENVIRONMENT     ASPNETCORE\_ENVIRONMENT
-ASPNET\_DETAILEDERRORS  ASPNETCORE\_DETAILEDERRORS
-======================  ===========================
+===================================================  ===================================================
+RC1                                                  RC2
+===================================================  ===================================================
+ASPNET_APP, ASPNET_APPLICATIONNAME, Hosting:App      ASPNETCORE_APPLICATIONNAME
+ASPNET_STARTUPASSEMBLY                               ASPNETCORE_STARTUPASSEMBLY
+ASPNET_DETAILEDERRORS, Hosting:DetailedErrors        ASPNETCORE_DETAILEDERRORS
+ASPNET_ENVIRONMENT, ASPNET_ENV, Hosting:Environment  ASPNETCORE_ENVIRONMENT
```

Comment:
We still support `ASPNET_ENV`, `Hosting:Environment` in RC2 but the user will see a message indicating these values are deprecated.